src/inferenceresponse.cpp

```diff
@@ -0,0 +1,61 @@
+//*****************************************************************************
+// Copyright 2022 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include "inferenceresponse.hpp"
+
+#include <string>
+#include <unordered_map>
+
+#include "inferenceparameter.hpp"
+#include "inferencetensor.hpp"
+#include "modelversion.hpp"
```

Comment:
Why do we need modelversion here if we do not use it anyway? (I suppose it was for model_version_t, but we use uint64_t anyway)